### PR TITLE
expose instead of ports

### PR DESCRIPTION
**Pull request by ciolansteen**
Changed yaml files to use expose instead port forwarding. Tested on DO and everything works

### CODE REVIEW

**Comment by akariv** [comment]
@ciolansteen 

1. Please make sure you're changing services that are part of the ckan dependency tree.
2. How would you connect to the ckan service after this change?
3. Please update documentation based on your modifications.

**Comment by ciolansteen** [comment]
1 - Checked communication between services using expose instead of ports publishing:
     varnish (depends on ckan)
     redis (ckan dependency)
     nginx (depends on ckan - for testing purposes, the port can be forwarded - included in troubleshooting.md)
     solr (ckan dependency)
     datapusher (not depending nor dependency)

Don't know how to test the services that have different container port from host port.
   *jenkins
   *cca-operator
   *provisioning-api
Please confirm if it is ok for these services to remain as is. 
     
2 - connecting to services can be done via ssh to the machine where docker is. There's no need to leave other ports open to the host since traefik takes care of exposing 443 and 80.

**Comment by akariv** [comment]
1. yes, leave as is
2. how would you connect to ckan service (not ssh, http)?
    is traefik exposing any ports at the moment?

**Comment by ciolansteen** [comment]
yes, both 80 and 443 are exposed by traefik

**Comment by akariv** [comment]
It doesn't look like it in the docker-compose file...

**Comment by ciolansteen** [comment]
it was in .docker-compose.vital-strategies-theme.yaml. Now is in .docker-compose-db.yaml

File: .docker-compose-db.yaml [hunk 1]
```diff
@@ -31,8 +31,8 @@ services:
   datastore-db:
     image: viderum/ckan-cloud-docker:db-latest
     restart: always
-    expose:
-    - "5432"
+    ports:
```
Comment (akariv) [inline .docker-compose-db.yaml]: why are we reverting this @ciolansteen ?

**Review by akariv — CHANGES_REQUESTED** [review]
See my comment - seems you last commit added more changes than you intended.

**Pull request closed without merging**